applications/serial_lte_modem/src/slm_ppp.c

```diff
@@ -489,20 +537,15 @@ static int handle_at_ppp(enum at_cmd_type cmd_type, const struct at_param_list *
 	} else if (op >= OP_COUNT) {
 		return -EINVAL;
 	}
-	const bool is_running = slm_ppp_is_running();
 
-	if ((!is_running && op == OP_STOP) || (is_running && op == OP_START)) {
-		return -EALREADY;
-	}
+	/* Send "OK" first in case stopping PPP results in the CMUX AT channel switching. */
+	rsp_send_ok();
```

Comment:
So we'll always send `OK` even if PPP won't change state (e.g. starting or stopping when it's already started or stopped)... Well, I guess it's okay.

Reply:
We could check those, but they would not be accurate as starting and stopping goes through the workqueue. The unsolicited notification will tell you when the state really changes.

Reply:
Yeah true. It's okay then.